vendor/github.com/google/go-github/github/github.go

```diff
@@ -17,6 +17,7 @@ import (
 	"io/ioutil"
 	"net/http"
 	"net/url"
+	"os"
```

Comment:
We don't want to make changes to this file as the vendor folder is automatically updated when we update dependencies. 

It appears the library already supports github enterprise in another way: https://github.com/google/go-github/blob/a83ae98ad5d09188c49d6056edb60ec9bdf202bd/github/github.go#L267

Reply:
Gotcha, thanks!